LWCE_Core/Src/XComLongWarCommunityEdition/Classes/LWCETacticalVisibilityHelper.uc

```diff
@@ -296,7 +296,7 @@ protected function HideAllVisibilityMarkers()
 
     foreach AllActors(class'XGUnit', kUnit)
     {
-        MarkUnit(kUnit, false, /* bFlanked */, kActiveUnit == kUnit);
+        MarkUnit(kUnit, false, false /* bFlanked */, kActiveUnit == kUnit);
```

Comment:
Style nitpick: the comment should be before the value, eg `/* bFlanked */ false`. In this case I left it empty because the value doesn't matter, due to the units not being visible, but I don't mind if you prefer to populate it.